lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -213,6 +213,8 @@ static int z_to_nrf_optname(int z_in_level, int z_in_optname,
 		case SO_REUSEADDR:
 			*nrf_out_optname = NRF_SO_REUSEADDR;
 			break;
+
+		/* Deprecated */
 		case SO_RAI_LAST:
 			*nrf_out_optname = NRF_SO_RAI_LAST;
 			break;
```

Comment:
add deprecation to all:
```suggestion
			break;
			/* Deprecated */
```

Reply:
Thank you for the feedback, I reworded it.